docker/webui/Dockerfile-lb

```diff
@@ -1,9 +1,8 @@
+#!BuildTag: openqa_webui_lb
```

Comment:
Please drop the second Dockerfile here as well, and this can be figured out in the future if/when necessary

Reply:
Yes, It is better to split this PR removing this Dockerfile from here. I created a new PR that contains the Dockerfile for the load balancer https://github.com/os-autoinst/openQA/pull/3544